churchland_pipeline/acquisition.py

```diff
@@ -0,0 +1,147 @@
+import datajoint as dj
+from . import lab, ephys, reference
+
+schema = dj.schema('churchland_acquisition')
+
+# -------------------------------------------------------------------------------------------------------------------------------
+# LEVEL 0
+# -------------------------------------------------------------------------------------------------------------------------------
+
+@schema
+class Session(dj.Manual):
+    definition = """
+    # Recording session
+    session_date: date # session date
+    -> lab.Monkey
+    ---
+    -> lab.User
+    -> [nullable] lab.User.proj(user2 = 'user') # secondary experimenter
```

Comment:
For multiple experimenters, their list may need to be normalized into a separate `Session.Experimenter` part table.